### PR TITLE
aTask3c

### PR DESCRIPTION
**Pull request by bradleypurdue**
https://trello.com/c/9ZR188RL/229-3c-update-view-to-accommodate-dropdown-with-view-helper

### CODE REVIEW

File: templates/displayApplicants.phtml [hunk 1]
```diff
@@ -10,15 +10,22 @@
...
     <script defer src="./js/deleteApplicant.js"></script>
+    <script defer src="./js/displayApplicants.js"></script>
 </head>
 <body>
 <div class="container">
     <div class="row">
         <div class="col-xs-12 registeredApplicants">
             <h1>Registered Applicants</h1>
+            <div class="col-4"><label for="cohorts">Select Cohort</label>
+            </div>
```
Comment (turnerror) [inline templates/displayApplicants.phtml:24]: This needs to be removed.

File: public/js/populateDropdowns.js [hunk 2]
```diff
@@ -27,7 +27,7 @@ let outputCohorts = async () => {
         cohortOptions += `value="${item.id}">${item.date}</option>`;
     });
 
-    element.innerHTML = cohortOptions;
+    element.innerHTML += cohortOptions;
 };
 
 let outputHearAbout = async () => {
```
Comment (alexjennings56) [inline public/js/populateDropdowns.js:33]: can this be a const? sorry just can't see it being re-assigned unless this is the reassignment? Just ignore if i'm wrong.

File: templates/displayApplicants.phtml [hunk 3]
```diff
@@ -10,15 +10,23 @@
...
     <div class="row">
         <div class="col-xs-12 registeredApplicants">
             <h1>Registered Applicants</h1>
             <form class="arrowForm" method="get">
+				<label for="cohorts">
+					Select Cohort
+					<select id="cohorts" class="submitApplicant form-control" name="cohortId">
+						<option value="all">All</option>
+					</select>
+				</label>
```
Comment (alexjennings56) [inline templates/displayApplicants.phtml:29]: indentation seems a bit off here? unless this is the correct way?

Comment (bradleypurdue) [reply templates/displayApplicants.phtml:29]: I think this is GitHub being funny, looks fine in PHPStorm.

File: public/js/populateDropdowns.js (hunk 2 above)
Comment (bradleypurdue) [reply public/js/populateDropdowns.js:33]: This isn't our code so no clue.

**Pull request merged**